migrations/1597041047603-CreateUsers.ts

```diff
@@ -0,0 +1,15 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
+
+export class CreateUsers1597041047603 implements MigrationInterface {
+    name = 'CreateUsers1597041047603'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query("CREATE TABLE `users` (`id` int NOT NULL AUTO_INCREMENT, `name` varchar(255) NOT NULL, `password` varchar(255) NOT NULL, `email` varchar(255) NOT NULL, UNIQUE INDEX `email` (`email`), PRIMARY KEY (`id`)) ENGINE=InnoDB");
+    }
```

Comment:
May be better to use migration api, easy too understand and code looks good, something like this
```
await queryRunner.createTable(
  new Table({
    name: "question",
    columns: [
      {
        name: "id",
        type: "int",
        isPrimary: true,
      },
      {
        name: "name",
        type: "varchar",
      },
    ],
  }),
  true
);

await queryRunner.createIndex(
  "question",
  new TableIndex({
    name: "IDX_QUESTION_NAME",
    columnNames: ["name"],
  })
);